src/Backend/Modules/FormBuilder/Actions/Edit.php

```diff
@@ -97,20 +97,7 @@ private function loadForm(): void
         $this->form->addText('textbox_validation_parameter');
         $this->form->addText('textbox_error_message');
 
-        // map the values to replace them with the backend translations
-        // use array combine to set the keys as the autocomplete values instead of key values
-        $this->form->addDropdown(
-            'textbox_autocomplete',
-            array_map(
-                function (string $value): string {
-                    return $value . ' (' . BL::getLabel('Autocomplete_' . str_replace('-', '_', $value)) . ')';
-                },
-                array_combine(
-                    Autocomplete::POSSIBLE_VALUES,
-                    Autocomplete::POSSIBLE_VALUES
-                )
-            )
-        );
+        $this->form->addDropdown('textbox_autocomplete', Autocomplete::getAutocompleteValues());
```

Comment:
As far as I can tell, the `getAutocompleteValues()` method isn't implemented yet so you might want to push that to git :) Also, `getAutocompleteValues()` is not a very good name since it's not very specific. `getValuesForDropdown()` might be a better one.